src/instrumenter/annotations.ts

```diff
@@ -502,20 +502,48 @@ function processMacroAnnotations(
 
                 const localAliases = new Map(globalAliases);
 
-                const targets = [...resolveAny(name, scope, ctx.compilerVersion, true)];
-
-                if (targets.length !== 1) {
-                    throw new MacroError(
-                        `No target ${name} found in contract ${
-                            (scope as ContractDefinition).name
-                        } for ${meta.original}`,
-                        meta.original,
-                        meta.parsedAnnot.src as Range,
-                        meta.target
-                    );
-                }
+                let target: AnnotationTarget;
 
-                const target = targets[0];
+                if (name === "<contract>") {
```

Comment:
I think this entire `if` block can be separated to a helper function with earlier returns. Otherwise `processMacroAnnotations()` becaming complex with e lot of nesting. It would be more readable if we keep it less complex.

Reply:
Fixed in 2e5e76b.